### PR TITLE
[DataObject] Classificationstore + QuantityValue not properly restored

### LINKED ISSUE #8114: [DataObject] Classificationstore + QuantityValue not properly restored
Regression of https://github.com/pimcore/pimcore/pull/6965

### PR DESCRIPTION
**Pull request by weisswurstkanone**
resolves #8114

### CODE REVIEW

**Review by dvesh3 — CHANGES_REQUESTED** [review]
Data not saved for direct inputQuantity fields on object class as generated tables unit column is bigint and it should be varchar
image

line: https://github.com/pimcore/pimcore/blob/48102717faaa6e0283644636457c9cdb77961c25/models/DataObject/ClassDefinition/Data/QuantityValue.php#L95

https://github.com/pimcore/pimcore/blob/48102717faaa6e0283644636457c9cdb77961c25/models/DataObject/ClassDefinition/Data/QuantityValue.php#L95

**Review by dvesh3 — CHANGES_REQUESTED** [review]
if InputQuantityValue field is inside Classificationstore, then on save it throws validation error..
image

**Comment by weisswurstkanone** [comment]
@dvesh3 unable to reproduce the validation error

File: bundles/CoreBundle/Migrations/Version20210211152319.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+     *
+     * @throws \Exception
+     */
+    public function up(Schema $schema)
+    {
+        $list = new ClassDefinition\Listing();
+        $list = $list->load();
+        foreach ($list as $class) {
+            $this->writeMessage(sprintf('Saving php files for class: %s', $class->getName()));
+            $class->generateClassFiles(false);
```
Comment (dvesh3) [inline bundles/CoreBundle/Migrations/Version20210211152319.php:28]: generating classes is not enough.....we need to up db tables as well for column type update

**Review by dvesh3 — APPROVED** [review]

**Review by dvesh3 — CHANGES_REQUESTED** [review]
Update migration

File: bundles/CoreBundle/Migrations/Version20210211152319.php (hunk 1 above)
Comment (dvesh3) [reply bundles/CoreBundle/Migrations/Version20210211152319.php:28]: Even calling `$class->save()` in migration has no impact on existing classes.

**Review by dvesh3 — APPROVED** [review]

**Pull request merged**